src/main/java/com/scalar/cassy/CassyBackupScheduler.java

```diff
@@ -0,0 +1,112 @@
+package com.scalar.cassy;
+
+import java.util.Optional;
+import java.util.concurrent.Callable;
+import picocli.CommandLine;
+
+@CommandLine.Command(
+    name = "cassy-schedule",
+    mixinStandardHelpOptions = true,
+    description = "calls Cassy's backup command",
+    subcommands = {ClusterSnapshot.class, NodeSnapshot.class, NodeIncremental.class})
+public class CassyBackupScheduler implements Runnable {
+
+  @CommandLine.Spec CommandLine.Model.CommandSpec spec;
+
+  public static void main(String[] args) {
+    System.exit(new CommandLine(new CassyBackupScheduler()).execute(args));
+  }
+
+  /**
+   * When an object implementing interface <code>Runnable</code> is used to create a thread,
+   * starting the thread causes the object's <code>run</code> method to be called in that separately
+   * executing thread.
+   *
+   * <p>The general contract of the method <code>run</code> is that it may take any action
+   * whatsoever.
+   *
+   * @see Thread#run()
+   */
+  @Override
+  public void run() {
+    spec.commandLine().usage(System.err);
+  }
+}
+
+@CommandLine.Command(name = "cluster_snapshot", aliases = {"cs"},description = "take a cluster-wide snapshot")
+class ClusterSnapshot implements Callable<Integer> {
+  CassyClient client = new CassyClient();
```

Comment:
To facilitate the testing afterwards, let's use Dependency Injection in this project like we are doing in a typical spring API application. 
As Cassy is already Guice as DI framework, I suggest to use it as well. 

If you want a sample on how to use it besides Cassy, you can refer to this PR. 
https://github.com/scalar-labs/indetail/pull/559/files
To sum up how to use Guice, all the dependencies are defined in a CommandModule class : https://github.com/scalar-labs/indetail/blob/audit_client_poc/ScalarDLAudit/src/main/java/com/scalar/audit/command/CommandModule.java  
Then you just need to add the `@Inject` annotation on top of your constructor for the dependency to be injected.
https://github.com/scalar-labs/indetail/blob/3325d8534c806237a694a8c2505fa511f7ed11f2/ScalarDLAudit/src/main/java/com/scalar/audit/command/Init.java#L39